src/app/index.ts

```diff
@@ -196,6 +196,9 @@ module.exports = class extends yo {
         }
       ];
       const answerForScriptType = await this.prompt(askForScriptType);
+      if (!answerForScriptType.scriptType && !this.options.ts) {
```

Comment:
We wanted to leave these off because the default should be set by the question settings.

Reply:
I'm not seeing that in the documentation and if it really didn't have the default answer then wouldn't there be an error when trying to create the project since there wouldn't be an answer (i.e. something like a null exception)?